src/_cffi_src/build_openssl.py

```diff
@@ -46,7 +46,7 @@ def _extra_compile_args(platform):
     When we drop support for CRYPTOGRAPHY_OPENSSL_LESS_THAN_110 we can
     revisit this.
     """
-    if platform != "win32":
+    if not platform in [ "win32", "hp-ux11" ]:
```

Comment:
To make pep8 happy this line should read `if platform not in ["win32", "hup-ux11"]:` (reordered not in and removed spaces in the list).